litmus/PTX/Manual/MP-3-transitive.litmus

```diff
@@ -0,0 +1,11 @@
+PTX MP-3-transitive
+"Adapted from Figure 6 of paper: Heterogeneous-race-free Memory Models"
+{
+x=0; y=0; z=0;
+P2:r0=0; P2:r1=0; P2:r2=0;
+}
+ P0@cta 0,gpu 0         | P1@cta 0,gpu 0         | P2@cta 0,gpu 1         ;
```

Comment:
I think it should be `P2@cta 1,gpu 0`.
The release-acquire sequence between `P1` and `P2` has `gpu` scope so both threads must be in the same `gpu 0` (and we need to palce them in different `cta` otherwise the `gpu` scope would not be necessary)

Reply:
Yes, updated.